mod-source-record-storage-server/src/main/java/org/folio/services/RecordSearchParameters.java

```diff
@@ -0,0 +1,88 @@
+package org.folio.services;
+
+import org.folio.rest.jaxrs.model.MarcRecordSearchRequest;
+import org.folio.rest.jaxrs.model.Record;
+
+/**
+ * Parameters needed for extended search
+ *
+ * @see MarcRecordSearchRequest
+ */
+public class RecordSearchParameters {
```

Comment:
Using Lombok will help to decrease the amount of boilerplate code. Also will be great to have equals and hashcode overridden.

Reply:
@KaterynaSenchenko  does it make sense to use the Lombok library for this class to make it pretty?

Reply:
@Igor-Gorchakov, I have no objections, you can use Lombok